ros/src/computing/perception/detection/packages/image_segmenter/include/image_segmenter_enet.hpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * image_segmenter_enet.hpp
+ *
+ *  Created on: Aug 23, 2017
+ *      Author: ne0
+ */
+
+#ifndef ENET_IMAGE_SEGMENTER_HPP_
+#define ENET_IMAGE_SEGMENTER_HPP_
+
+#include <caffe/caffe.hpp>
+
+#include <opencv2/core/core.hpp>
+#include <opencv2/highgui/highgui.hpp>
+#include <opencv2/imgproc/imgproc.hpp>
+
+#include <algorithm>
+#include <iosfwd>
+#include <memory>
+#include <string>
+#include <utility>
+#include <vector>
```

Comment:
see google style guide for order of includes. cafffe and opencv should go after system libs.